whisper/whisperv6/peer_test.go

```diff
@@ -101,7 +101,7 @@ var debugMode = false
 var prevTime time.Time
 var cntPrev int
 
-func TestSimulation(t *testing.T) {
```

Comment:
@justelad we have to revert this on our end. Whisper team most probably has already fixed this.